src/index.html

```diff
@@ -9,6 +9,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img 
+          src="images/logo.png" 
+          alt="Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link  is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
      <a href="#" class="logo">
        <img 
          src="images/logo.png" 
          alt="Logo"
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link  is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
```
fix code style all over the page